src/systems/audio-system.js

```diff
@@ -61,20 +61,33 @@ export class AudioSystem {
       evt.detail.cameraEl.getObject3D("camera").add(sceneEl.audioListener);
     });
 
+    const ctx = THREE.AudioContext.getContext();
+    this.outboundContext = new (window.AudioContext || window.webkitAudioContext)();
+    this.audioNodes = new Map();
+
     /**
      * Chrome and Safari will start Audio contexts in a "suspended" state.
      * A user interaction (touch/mouse event) is needed in order to resume the AudioContext.
      */
-    const ctx = THREE.AudioContext.getContext();
     const resume = () => {
       ctx.resume();
+      this.outboundContext.resume();
 
-      setTimeout(function() {
-        if (ctx.state === "running") {
+      setTimeout(() => {
+        if (ctx.state === "running" && this.outboundContext.state === "running") {
           if (!AFRAME.utils.device.isMobile() && /chrome/i.test(navigator.userAgent)) {
             enableChromeAEC(sceneEl.audioListener.gain);
           }
 
+          this.mediaStreamDestinationNode = this.outboundContext.createMediaStreamDestination();
+          this.outboundStream = this.mediaStreamDestinationNode.stream;
+          this.outboundGainNode = this.outboundContext.createGain();
+          this.outboundAnalyser = this.outboundContext.createAnalyser();
+          this.outboundAnalyser.fftSize = 32;
+          this.analyserLevels = new Uint8Array(this.outboundAnalyser.fftSize);
+          this.outboundGainNode.connect(this.outboundAnalyser);
+          this.outboundAnalyser.connect(this.mediaStreamDestinationNode);
+
```

Comment:
Not related to this commit but we avoid the call to `removeEventListener` below by passing `once:true` as `options` to `addEventListener`:
```
document.body.addEventListener("touchend", resume, {capture: false, once: true});
```
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters